README.md

```diff
@@ -193,124 +242,119 @@ Other acceptable location examples:
 
-Our probes don't expose any unique IDs to target. The recommended way to narrow down your results is to use filters and even stack them together.
+### Leverage system tags (eyeball networks and more) 🏷️
+Many probes are going to be tagged by our system. At the moment, this includes:
 
-This way popular locations such as `from Amsterdam` or `from AWS` will be automatically load-balanced among multiple probes in the same location and avoid overloading any specific one.
+- **Google Cloud and AWS cloud region names**. For example, `aws-eu-west-1` and `gcp-us-south1`. These tags follow the respective provider's naming scheme and are prefixed with their name.
+- **Eyeball and data center network tags**. `eyeball` and `datacenter`. `eyeball` probes are hosted with ISPs that provide internet access to regular people and small businesses. As the name suggests, `datacenter` tags are intended for probes hosted in a data center.
 
-### Best effort results
+### Things to keep in mind
 
-When requesting a specific number of probes there is no guarantee that the API will respond back with the exact same amount. Keep this in mind and use it to your benefit.
+#### Probes share no UUIDs
+Our probes don't expose unique IDs that you can use to target them explicitly. Our probes don't expose unique IDs that you can use to target them explicitly. Instead, we recommend fine-tuning the probe selection by using and combining filters or specifying IDs from previous measurements, as described in the best practice section above.
 
-### Probe availability
+This ensures that popular locations, like `from Amsterdam` or `from AWS`, are automatically load-balanced across multiple probes within the same location, preventing the overload of specific probes.
 
-Globalping only exposes and allows users to interact with online probes. A probe is considered online and becomes available via our API when:
+#### Test result variability
+When requesting a specific number of probes, there is no guarantee that the API will respond with the exact amount.
 
-- It's up-to-date. A probe running an older version will be forced to auto-update before it becomes available.
-- Passes quality control tests. Every probe must first pass a test to ensure it's connected to a reasonably stable network with no packet loss. This test is then repeated on a schedule and will take a probe offline if it fails the QA test.
+#### Probe availability rules
+Globalping exposes and lets you interact with probes that are currently **online**. 
 
-This is completely automated and managed by our platform.
+Here's when a probe is considered online and available through our API:
+- **It's up-to-date.** A probe running an older version will be forced to auto-update before it becomes available.
+- **It passes quality control tests.** Every probe must pass a test, ensuring it's connected to a reasonably stable network without packet loss. This test is done regularly; if a probe fails, it gets taken offline.
```

Comment:
Lets add:
- It's not a VPN. We block probes that we detect as being hosted behind a VPN or any other proxying technique. Such probes would report incorrect latency and routing data making them unusable.